apps/desktop/scripts/finalize-renderer-assets.js

```
@@ -0,0 +1,41 @@
+async function postBuild() {
+  try {
+    // The old script did `mv ./web-build ./app/build`.
+    // We need to move the webpack output from web-build to app/build
+    if (await fs.pathExists(webBuildDir)) {
+      console.log(`Moving ${webBuildDir} to ${appBuildDir}...`);
+      // Remove existing app/build if it exists
+      await fs.remove(appBuildDir);
+      // Move web-build to app/build
+      await fs.move(webBuildDir, appBuildDir);
+    } else {
+      console.error(`Error: Source directory ${webBuildDir} does not exist. Webpack build might have failed.`);
```

ESLint

[error] Replace ``Error:·Source·directory·${webBuildDir}·does·not·exist.·Webpack·build·might·have·failed.`` with `⏎········`Error:·Source·directory·${webBuildDir}·does·not·exist.·Webpack·build·might·have·failed.`,⏎······`

(prettier/prettier)